wallet/psbt.go

```diff
@@ -176,11 +178,23 @@ func (w *Wallet) FundPsbt(packet *psbt.Packet, keyScope *waddrmgr.KeyScope,
 		}
 		inputSource := constantInputSource(credits)
 
+		// Build the TxCreateOption to retrieve the change scope
+		opts := defaultTxCreateOptions()
+		for _, optFunc := range optFuncs {
+			optFunc(opts)
+		}
+
+		// If the change scope isn't set, then it should be the same as the
+		// coin selection scope in order to match existing behavior.
```

Comment:
"existing behavior", does that still refer to something meaningful after this is merged?

Reply:
Actually yes.
Before this PR we passed `keyScope` as the `changeScope`.
Maybe "previous behaviour" is better?

Reply:
What I mean is that a reader of this comment can't know what the previous behavior was. It's more appropriate in the commit message perhaps.